app/src/main/res/anim/fade_in.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<alpha xmlns:android="http://schemas.android.com/apk/res/android"
+       android:interpolator="@android:anim/accelerate_interpolator"
+       android:fromAlpha="0.0" android:toAlpha="1.0"
+       android:duration="500" />
```

Comment:
I recommend moving the durations to dimens.xml so that we can keep the fade-in/fade-out times consistent here & below.

Reply:
moved durations to dimens.xml